homeassistant/components/spotify/__init__.py

```diff
@@ -60,7 +61,9 @@ async def async_browse_media(
     hass, media_content_type, media_content_id, *, can_play_artist=True
 ):
     """Browse Spotify media."""
-    info = list(hass.data[DOMAIN].values())[0]
+    info = next(iter(hass.data[DOMAIN].values()), None)
+    if not info:
+        raise BrowseError("No Spotify accounts available")
```

Comment:
Should we catch this inside Sonos?

Reply:
I think this makes a lot of sense to do here, mainly because anything else can just use it without having to worry about it.

Reply:
Sorry, I meant catch `BrowseError` in Sonos. The check here is indeed correct.

Reply:
I don't think that's necessary. It should only hit this codepath if the browser is explicitly trying to browse into Spotify. It'll only hit this at the root level is Spotify is the only source available.